lib/ecommerce/resources/collection.rb

```diff
@@ -11,7 +11,7 @@ class Collection < Base
       def initialize(response)
         @response = response
         @collection = []
-        @headers = response.headers['Link'].split(',')
+        @headers = response.headers['Link'].split(',') if response.headers['Link']
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.